sympy/core/compatibility.py

```diff
@@ -92,7 +92,7 @@ def u_decode(x):
 
     exec_ = getattr(builtins, "exec")
 
-    xrange = range
+    range = range
```

Comment:
This should remain as it was. If xrange is used it should be replaced with range in PY3 since there is no xrange in PY3.